unblob/extractor.py

```diff
@@ -86,7 +93,7 @@ def fix_symlink(path: Path, outdir: Path, task_result: TaskResult) -> Path:
 
 def fix_extracted_directory(outdir: Path, task_result: TaskResult):
     fix_permission(outdir)
-    for path in outdir.rglob("*"):
+    for path in (Path(p) for p in glob.iglob(f"{outdir}/**", recursive=True)):
```

Comment:
There is a regression problem here compared to the previous version: `Path.glob()` returned hidden files (those, that start with `.`), while `glob.*glob` does not.

Looking at the `glob` module doc, there is a parameter `include_hidden`, but it was added only in 3.11!!!

I think the only remaining option besides going still lower-level is using `os.walk`.

Reply:
Fix and detailed explanation in commit abbc974